target/linux/kirkwood/base-files/etc/board.d/02_network

```diff
@@ -26,8 +26,8 @@ case "$board" in
 	;;
 "linksys,audi"|\
 "linksys,viper")
-	ucidef_add_switch "switch0" \
-		"0:lan" "1:lan" "2:lan" "3:lan" "4:wan" "5@eth0" "6@eth1"
+	ucidef_set_interfaces_lan_wan "ethernet1 ethernet2 ethernet3 ethernet4" "internet"
```

Comment:
This really hurts my eyes. I'd rather patch the DTS to use lan1-lan4 and wan ... (not your fault of course).

Other people having an opinion on this one?

Reply:
The question is: Should  we be close to upstream or make it proper? 
I agree, lan1-4/wan names are better.

Reply:
I will write an e-mail to the devel list this evening, and try to answer this in general. We have several targets moving at the moment, so this might be a good point to discuss.